src/main/java/mate/academy/util/PasswordUtil.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class PasswordUtil {
+    private static final String HASH_ALGORITHM = "SHA-256";
+    private static final String HEX_STRING = "%02x";
+    private static final int SALT_LENGTH = 16;
+    private static final SecureRandom secureRandom = new SecureRandom();
```

Comment:
i think better to initialize in method

Reply:
I'm not agree with that. Why should i create every time new instance, when method called, if i can initialize it once and reuse. Also there are other possible drawbacks of creating new instance of SecureRandom every time. https://stackoverflow.com/questions/295628/securerandom-init-once-or-every-time-it-is-needed